Formula/py3cairo.rb

```diff
@@ -13,7 +14,7 @@ class Py3cairo < Formula
 
   depends_on "pkg-config" => :build
   depends_on "cairo"
-  depends_on "python"
+  depends_on "python@3.8"
 
   def install
     system "python3", *Language::Python.setup_install_args(prefix)
```

Comment:
Python 3.7 is getting dragged in here from somewhere.

Reply:
Because of `glib` and `cairo`. I will bump some more revisions in this PR.

Reply:
`glib` is awkward as it has a very large dependency tree. Python isn't actually used at build time in `glib`, when I last checked anyway, so there was an attempt to deal with it separately in #47447.

That was before we had shebang rewrite stuff however, so some of the concerns of that pull request no longer apply.